spec/services/csv_exporter_spec.rb

```diff
@@ -28,12 +26,12 @@
                              'FOI Responding Team',
                              'foi responding user',
                              '2018-08-30',
-                             '2018-09-13',
+                             nil,
```

Comment:
I think we should be changing the test data so that these columns retain the same values. Or adding more tests so that all the cases are covered?